x-pack/plugins/alerting/public/pages/maintenance_windows/translations.ts

```diff
@@ -428,3 +428,116 @@ export const TABLE_START_TIME = i18n.translate(
 export const TABLE_END_TIME = i18n.translate('xpack.alerting.maintenanceWindows.table.endTime', {
   defaultMessage: 'End time',
 });
+
+export const TABLE_ACTION_EDIT = i18n.translate('xpack.alerting.maintenanceWindows.table.edit', {
+  defaultMessage: 'Edit',
+});
+
+export const EDIT_MAINTENANCE_WINDOW = i18n.translate(
+  'xpack.alerting.maintenanceWindows.edit.maintenanceWindow',
+  {
+    defaultMessage: 'Edit maintenance window',
+  }
+);
+
+export const SAVE_MAINTENANCE_WINDOW = i18n.translate(
+  'xpack.alerting.maintenanceWindows.save.maintenanceWindow',
+  {
+    defaultMessage: 'Save maintenance window',
+  }
+);
+
+export const TABLE_ACTION_CANCEL = i18n.translate(
+  'xpack.alerting.maintenanceWindows.table.cancel',
+  {
+    defaultMessage: 'Cancel',
+  }
+);
+
+export const CANCEL_MODAL_TITLE = i18n.translate(
+  'xpack.alerting.maintenanceWindows.cancelModal.title',
+  {
+    defaultMessage: 'Cancel maintenance window',
+  }
+);
+
+export const CANCEL_MODAL_SUBTITLE = i18n.translate(
+  'xpack.alerting.maintenanceWindows.cancelModal.subtitle',
+  {
+    defaultMessage:
+      'Rule notifications resume immediately. Running maintenance window events are canceled.',
+  }
+);
+
+export const CANCEL_MODAL_BUTTON = i18n.translate(
+  'xpack.alerting.maintenanceWindows.cancelModal.button',
+  {
+    defaultMessage: 'Keep running',
+  }
+);
+
+export const TABLE_ACTION_CANCEL_AND_ARCHIVE = i18n.translate(
+  'xpack.alerting.maintenanceWindows.table.cancelAndArchive',
+  {
+    defaultMessage: 'Cancel and Archive',
+  }
+);
+
+export const CANCEL_AND_ARCHIVE_MODAL_TITLE = i18n.translate(
+  'xpack.alerting.maintenanceWindows.cancelAndArchiveModal.title',
+  {
+    defaultMessage: 'Cancel and archive maintenance window',
+  }
+);
+
+export const CANCEL_AND_ARCHIVE_MODAL_SUBTITLE = i18n.translate(
+  'xpack.alerting.maintenanceWindows.cancelAndArchiveModal.subtitle',
+  {
+    defaultMessage:
+      'Rule notifications resume immediately. Running and upcoming maintenance window events are canceled.',
```

Comment:
My understanding of the design doc was that "archived" maintenance windows would be deleted in X days, whereas "canceled" maintenance windows would linger indefinitely. If that's correct, I'd change it to something like this:
```suggestion
      'Rule notifications resume immediately. All running and upcoming events in the maintenance window are canceled and the window is queued for deletion.',
```